src/modules/commander/Commander.cpp

```diff
@@ -1070,13 +975,60 @@ Commander::handle_command(vehicle_status_s *status_local, const vehicle_command_
 		cmd_result = handle_command_motor_test(cmd);
 		break;
 
+	case vehicle_command_s::VEHICLE_CMD_PREFLIGHT_REBOOT_SHUTDOWN:
+
+		// do nothing for autopilot
+		if (((int)(cmd.param1)) == 0) {
+			answer_command(cmd, vehicle_command_s::VEHICLE_CMD_RESULT_ACCEPTED, command_ack_pub);
+
+			break;
+		}
+
+		if (shutdown_if_allowed()) {
+			bool shutdown_ret_val = PX4_ERROR;
+
+			if (((int)(cmd.param1)) == 1) {
```

Comment:
```suggestion
			if (shutdown_type == 1) {
```

Reply:
No, I think we should be using `roundf` everywhere for these params.